athina_client/datasets/dataset.py

```diff
@@ -33,9 +33,9 @@ def _check_forbidden_keys(rows: List[Dict[str, Any]]):
     def create(
         name: str,
         description: Optional[str] = None,
-        language_model_id: Optional[str] = None,
-        prompt_template: Optional[Any] = None,
```

Comment:
@shivsak These are top level items in dataset, should they be removed?

Reply:
Yes, we have deprecated these.

They are not in our documentation, or UI.